src/NuGet.ApplicationInsights.Owin/NuGet.ApplicationInsights.Owin.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" Condition="Exists('..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props')" />
```

Comment:
>MicroBuild.Core.0.3.0 [](start = 34, length = 21)

If we bump up the MicroBuild version, we'll have to hunt for all the places that reference the version. I'm worried this could be easy to mess up without realizing. Would it be possible to have a `MicroBuildVersion` property?

Reply:
This is done by NuGet client when you update the package.

Reply:
Yup, props and targets.